Cargo.toml

```diff
@@ -15,6 +15,7 @@ keywords = ["linear-algebra"]
 accelerate = ["accelerate-src"]
 netlib = ["netlib-src"]
 openblas = ["openblas-src"]
+intel-mkl = ["intel-mkl-src"]
```

Comment:
Let’s keep it sorted everywhere. Please put `intel-mkl` after `accelerate`.

Reply:
- accelerate
- intel-mkl
- openblas
- netlib

right?

Reply:
Yes, great!